api/krusty/testdata/repo_read_only_ssh_key.yaml

```diff
@@ -0,0 +1,5 @@
+# This is a base64 encoded SSH private key configured as a GitHub Deploy key,
+# with read-only access to the kustomize repo
```

Comment:
Does this key have an expiration date? If so, we should include in this comment that we will need to refresh it.